inc/datacollector/class-book.php

```diff
@@ -261,7 +275,15 @@ public function copyBookMetaIntoSiteTable( $book_id ) {
 		update_site_meta( $book_id, self::STORAGE_SIZE, $space_used );
 
 		// pb_language
-		update_site_meta( $book_id, self::LANGUAGE, $metadata['pb_language'] ?? 'en' );
+		$book_language = $metadata['pb_language'] ?? 'en';
+		update_site_meta( $book_id, self::LANGUAGE, $book_language );
+
+		$languages = \Pressbooks\L10n\supported_languages();
+		$language = ( array_key_exists( $book_language, $languages ) ) ?
+			$languages[ $book_language ] : 'English';
+
+		// pb_language_name
+		update_site_meta( $book_id, self::LANGUAGE_NAME, $language );
```

Comment:
What if we use the null coalescing here as well?
```suggestion
		// pb_language_name
		update_site_meta( $book_id, self::LANGUAGE_NAME, $languages[ $book_language ] ?? 'English' );
```